libraries/ESP8266WebServer/examples/HttpHashCredAuth/HttpHashCredAuth.ino

```diff
@@ -0,0 +1,258 @@
+/*
+  HTTP Hashed Credential example
+  Created April 27, 2019 by Tyler Moore.
+  This example code is in the public domain.
+
+  This is a simple Arduino example to demonstrate a few simple techniques:
+  1. Creating a secure web server using ESP8266ESP8266WebServerSecure
+  2. Use of HTTP authentication on this secure server
+  3. A simple web interface to allow an authenticated user to change Credentials
+  4. Persisting those credentials through a reboot of the ESP by saving them to SPIFFS without storing them as plain text
+*/
+
+#include <FS.h>
+#include <ESP8266WiFi.h>
+#include <ESP8266WebServerSecure.h>
+
+//Unfortunately it is not possible to have persistent WiFi credentials stored as anything but plain text. Obfuscation would be the only feasible barrier.
+#ifndef STASSID
+#define STASSID "your-ssid"
+#define STAPSK  "your-password"
+#endif
+
+const char* ssid = STASSID;
+const char* wifi_pw = STAPSK;
+
+const String file_credentials=R"(/credentials.txt)"; //SPIFFS file name for the saved credentials
```

Comment:
The CI system demands spacing here for Arduino astyle: `... file_credentials = R"(...`

Reply:
Thanks! I believe the new commit to this branch addresses all style issues.